scanner/src/test/assets/scanoss-junit-4.12.json

```diff
@@ -2,6 +2,22 @@
   "/tmp/ort-ScanOss2759786101559527642/Maven/junit/junit/4.12/to-do.txt": [
     {
       "id": "none",
+      "status": "pending",
```

Comment:
Why was this file modified? Where do these changes come from? Maybe do this in a separate preceding commit with more explanations in the commit message?